cosmicfishpie/version.py

```diff
@@ -1,8 +1,8 @@
-_MAJOR = "0"
+_MAJOR = "1"
 _MINOR = "1"
 # On main and in a nightly release the patch should be one ahead of the last
 # released build.
-_PATCH = "7"
+_PATCH = "0"
```

Comment:
yeah, I merged overview PR, so this should go on top. I added a new Minor version. Not sure why overview PR still had a zero in Major. We are already in 1.x since a while.